rnnalyse/extractors/base_extractor.py

```diff
@@ -86,15 +86,16 @@ def extract(self,
             be extracted or not.
         """
         start_t = prev_t = time()
-        num_extracted = n_sens = 0
+        n_extracted = n_sens = 0
```

Comment:
Is it necessary to initialize `n_sens` here? 🤔

Reply:
So in practice it isn't, but to be completely type-correct you should initialize a variable that is used in a for loop if you want to use it afterwards.

So omitting `n_sens` here wouldn't necessarily break anything, only when `self.corpus` is empty.